manifests/s/StartIsBack/StartIsBack/2.9.17/StartIsBack.StartIsBack.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: StartIsBack.StartIsBack
+PackageVersion: 2.9.17
+PackageLocale: en-US
+Publisher: www.startisback.com
+PackageName: StartIsBack

```

```suggestion
PackageName: StartIsBack++
```
This must match Control Panel.